test/liftN.js

```diff
@@ -50,4 +44,25 @@ describe('liftN', function() {
     var addM = R.liftN(2, R.add);
     assert.deepEqual(addM(Maybe(3), Maybe(5)), Maybe(8));
   });
+
+  it('correctly interprets [] as a functor', function() {
+    assert.deepEqual(addN3([1, 2, 3], [1, 2], [1, 2, 3]), [3, 4, 5, 4, 5, 6, 4, 5, 6, 5, 6, 7, 5, 6, 7, 6, 7, 8]);
+    assert.deepEqual(addN3([1], [2], [3]), [6]);
+    assert.deepEqual(addN3([1, 2], [3, 4], [5, 6]), [9, 10, 10, 11, 10, 11, 11, 12]);
+  });
+
+  it('correctly interprets ((->) r) as a functor', function() {
+    var convergedOnInt = addN3(R.add(2), R.multiply(3), R.subtract(4));
+    var liftedAnd = R.lift(R.and);
+    var convergedOnBool = liftedAnd(R.gt(R.__, 0), R.lt(R.__, 3));
```

Comment:
Let's remove the `liftedAnd` variable. `liftedAnd` doesn't convey any more information than `R.lift(R.and)`. ;)